editor/editor_node.cpp

```diff
@@ -5853,8 +5853,15 @@ EditorNode::EditorNode() {
 	add_editor_plugin(memnew(AnimationTreeEditorPlugin(this)));
 	add_editor_plugin(memnew(MeshLibraryEditorPlugin(this)));
 	add_editor_plugin(memnew(StyleBoxEditorPlugin(this)));
-	add_editor_plugin(memnew(SpriteEditorPlugin(this)));
-	add_editor_plugin(memnew(Skeleton2DEditorPlugin(this)));
+
+	// hide these plugins by default
+	SpriteEditorPlugin *sprite_editor_plugin = memnew(SpriteEditorPlugin(this));
+	Skeleton2DEditorPlugin *skeleton2d_editor_plugin = memnew(Skeleton2DEditorPlugin(this));
+	add_editor_plugin(sprite_editor_plugin);
+	add_editor_plugin(skeleton2d_editor_plugin);
+	sprite_editor_plugin->make_visible(false);
```

Comment:
Maybe those two should be hidden directly in their constructor, as done for most other editor plugins?